src/load-data.test.js

```diff
@@ -1,16 +1,75 @@
 'use strict'
 
 import { expect } from 'chai'
+import sinon from 'sinon'
 
 import loadDataIntoLocalStorage from './load-data'
 
+const SMALLER_SINCE = 0
+const LARGER_SINCE = 1
+
 describe('lib.load-data.loadDataIntoLocalStorage', () => {
-  it('should throw a not implemented error', () => {
-    try {
-      loadDataIntoLocalStorage({})
-    } catch (e) {
-      return expect(e.message).equals('Not implemented')
+  let localStorageOverride
+  beforeEach(() => {
+    localStorageOverride = {
+      getItem: sinon.stub(),
+      removeItem: sinon.stub(),
+      setItem: sinon.stub()
+    }
+  })
+  it('should not affect localStorage if its data is more recent', () => {
+    localStorageOverride.getItem.returns(LARGER_SINCE)
+
+    loadDataIntoLocalStorage({ serializedData: { since: SMALLER_SINCE } }, localStorageOverride)
+
+    expect(localStorageOverride.getItem.calledOnce).to.equal(true)
```

Comment:
[Sinon-chai](https://www.chaijs.com/plugins/sinon-chai/) adds some nice syntactic sugar for this.

In this case it would be:
```javascript
expect(localStorageOverride.getItem).to.have.been.calledOnce;
```

Reply:
That's going to fail `standard` linting, so we should avoid it.